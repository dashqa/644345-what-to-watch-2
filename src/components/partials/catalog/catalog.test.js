import React from 'react';
import Catalog from './catalog';
import renderer from "react-test-renderer";

const props = {
  movies: [],
  genres: {},
  activeFilter: ``,
  isMainPage: false,
  onChangeFilter: () => {}
};

it(`Catalog correctly renders after relaunch`, () => {
  const tree = renderer.create(<Catalog {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
