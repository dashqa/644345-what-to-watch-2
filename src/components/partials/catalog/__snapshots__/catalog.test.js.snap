// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Catalog correctly renders after relaunch 1`] = `
<section
  className="catalog"
>
  <h2
    className="catalog__title"
  >
    
  </h2>
  <div
    className="catalog__movies-list"
  />
</section>
`;
